### ISSUE TITLE
none: waiting for k8s-app=kube-proxy: timed out waiting for the condition

### ISSUE THREAD

**Issue by marcchernoff** [kubernetes/minikube#3980]
Ubuntu 18.04.1 LTS
Release:	18.04
Codename:	bionic

```
user@ubuntu:~/Downloads/minikube$ sudo minikube start --vm-driver=none
o   minikube v0.35.0 on linux (amd64)
>   Configuring local host environment ...
...
:   Restarting existing none VM for "minikube" ...
:   Waiting for SSH access ...
-   "minikube" IP address is 192.168.100.131
-   Configuring Docker as the container runtime ...
-   Preparing Kubernetes environment ...
-   Pulling images required by Kubernetes v1.13.4 ...
:   Relaunching Kubernetes v1.13.4 using kubeadm ...
:   Waiting for pods: apiserver proxy!   Error restarting cluster: wait: waiting for k8s-app=kube-proxy: timed out waiting for the condition
```

**Comment by marcosdiez** [comment]
Hi! I had a similar problem and my solution is here: https://github.com/kubernetes/minikube/pull/4052
Could you please try yourself and comment if the solution works for you as well ? Thank you!

**Comment by tstromberg** [comment]
Thanks for the bug report. Do you mind attaching the output of `minikube logs`?

**Comment by marcchernoff** [comment]
Hi @tstromberg - I can't get any logs.  Any command I run gives me:  "The connection to the server 192.168.100.131:8443 was refused - did you specify the right host or port?"
Any ideas?

Thanks!!!

**Comment by tstromberg** [comment]
I believe this issue was resolved in the v1.1.0 release. Please try upgrading to the latest release of minikube and run `minikube delete` to remove the previous cluster state.

If the same issue occurs, please re-open this bug. Thank you opening this bug report, and for your patience!
